spyder_kernels/utils/test_utils.py

```diff
@@ -19,10 +19,10 @@
 except ImportError:
     from io import StringIO
 
-from spyder_kernels.console.kernel import SpyderKernel
+from spyder_kernels.console.kernel import ConsoleKernel
 
 
-def get_kernel(kernel_class=SpyderKernel):
+def get_kernel(kernel_class=ConsoleKernel):
```

Comment:
This function should be called now `get_console_kernel`.

Reply:
Or does it work for both kernels?

Reply:
Works for both. As a parameter you give the class of the kernel you want to test.

Reply:
Ok, I see.